Dockerfile

```diff
@@ -15,7 +15,8 @@ ENV CONF_PHPFPM=/etc/php/7.0/fpm/php-fpm.conf \
     PHP_FPM_MEMORY_LIMIT=256M \
     PHP_FPM_MAX_EXECUTION_TIME=60 \
     PHP_FPM_UPLOAD_MAX_FILESIZE=1M \
-    NEWRELIC_VERSION=6.7.0.174
+    NEWRELIC_VERSION=6.7.0.174 \
+    CFG_APP_DEBUG=1
```

Comment:
Since it's not the greatest name but can't change it ... can you also put a comment in this Dockerfile (I see you did document in the README)?  @bryanlatten